src/server/csp.ts

```diff
@@ -121,6 +121,39 @@ const buildCsp = (
     media-src 'self' https://audio.guim.co.uk/
 `.trim();
 
+function cspEditions(
+	{ styles }: Assets,
+	thirdPartyEmbed: ThirdPartyEmbeds,
+): string {
+	return `
+	default-src 'self';
+	style-src 'self' ${assetHashes(styles)};
+	img-src 'self' https://static.theguardian.com https://*.guim.co.uk ${
+		thirdPartyEmbed.twitter
+			? 'https://platform.twitter.com https://syndication.twitter.com https://pbs.twimg.com data:'
+			: ''
+	};
+    script-src 'self' ${
+		thirdPartyEmbed.instagram ? 'http://www.instagram.com/embed.js' : ''
+	} https://interactive.guim.co.uk https://s16.tiktokcdn.com https://www.tiktok.com/embed.js https://sf16-scmcdn-sg.ibytedtos.com/ ${
```

Comment:
If Editions doesn't currently support Interactives, Instagram or TikTok (`ibytedtos` is a TikTok domain FYI) then I'd be inclined to drop these URLs until you _know_ you need them.